tools/stability-checker/README.md

```diff
@@ -16,7 +16,11 @@ export KUBECONFIG="{path to kubeconfig}"
 2. Provision volume with a testing script, which then will be used by the Stability Checker. You can use the `local/provision_volume.sh` script. 
 The script copies all files placed in the `local/input` directory to PV.
 
-3. Install the Stability Checker as a Helm chart and provide the proper configuration:
+3. Logs from stability checker by default are sent to Google Cloud Storage bucket, to make it works create secret
```

Comment:
IMO by default it should be disabled, to keep backward compatibility - this will require changes in the prowjob.